pkg/trafficpolicy/trafficpolicy.go

```diff
@@ -9,7 +9,6 @@ import (
 	hashstructure "github.com/mitchellh/hashstructure/v2"
 
 	policyv1alpha1 "github.com/openservicemesh/osm/pkg/apis/policy/v1alpha1"
-
```

Comment:
I think we still want this space here to separate locally generated k8s resources from app code

Reply:
done